components/forms/Login/index.jsx

```diff
@@ -0,0 +1,61 @@
+import Link from 'next/link';
+
+export const loginFormTestIds = {
+  form: 'login-form',
+  emailLabel: 'email',
+  emailField: 'input-email',
+  passwordLabel: 'password',
+  passwordField: 'input-password',
+  signInButton: 'btn-sign-in',
```

Comment:
Maybe we can make this a bit more consistent by using the term `login` instead of `sign in`, what do you think?

```suggestion
  loginButton: 'btn-login',
```

Reply:
sure, renamed all `signin` instances to `login` for consistency in https://github.com/tyrro/react-survey/pull/50/commits/ac569c09a1643adb280fe4d6a3705d1c347ab318